dev-python/protobuf-python/protobuf-python-3.19.1.ebuild

```
@@ -0,0 +1,52 @@
+# Copyright 2008-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Empty line after EAPI

```suggestion
EAPI=7

```